src/golang/lib/engine/databricks_executor.go

```diff
@@ -0,0 +1,144 @@
+package engine
+
+import (
+	"context"
+	"time"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	"github.com/aqueducthq/aqueduct/lib/job"
+	dag_utils "github.com/aqueducthq/aqueduct/lib/workflow/dag"
+	"github.com/aqueducthq/aqueduct/lib/workflow/operator"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// We seperate out the execution step for Databricks Jobs since
+// Databricks takes care of launching Tasks.
+// Steps:
+// 1. Convert each operator into a Task (includes parent dependency).
```

Comment:
so here the main difference is that we don't schedule downstreams when one op succeeded? And the dependencies are handled by databricks?

Reply:
Yes, while executing with Databricks Jobs, we don't schedule any individual operators. We package all of them in one single job and upload it to Databricks. Their task orchestrator takes care of launching tasks once the appropriate upstream operators are finished.